containers/Dockerfile.buildenv

```diff
@@ -131,7 +131,7 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.12.4/cmake-3.12.4
 # Build with GCC7 when easily possible to maximize ABI compatibility.
```

Comment:
[random line in this file] I also needed to change:
```
&& curl https://bootstrap.pypa.io/get-pip.py --output get-pip.py \ 
```
to
```
&& curl https://bootstrap.pypa.io/2.7/get-pip.py --output get-pip.py \ 
```
above. Could you add a commit at the end that does that?